<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>149</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>12</y>
      <width>507</width>
      <height>66</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>69</width>
             <height>26</height>
            </size>
           </property>
           <property name="text">
            <string>Threshold</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="slider_threshold">
           <property name="minimumSize">
            <size>
             <width>190</width>
             <height>16</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>190</width>
             <height>16</height>
            </size>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>120</number>
           </property>
           <property name="value">
            <number>20</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_threshold">
           <property name="maximumSize">
            <size>
             <width>48</width>
             <height>26</height>
            </size>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>120</number>
           </property>
           <property name="value">
            <number>20</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>41</width>
             <height>26</height>
            </size>
           </property>
           <property name="text">
            <string>Proba(/100)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="slider_proba">
           <property name="minimumSize">
            <size>
             <width>190</width>
             <height>16</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>190</width>
             <height>16</height>
            </size>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>5</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_proba">
           <property name="maximumSize">
            <size>
             <width>48</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>5</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QRadioButton" name="radio_plane">
         <property name="text">
          <string>Plane</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radio_cloud">
         <property name="text">
          <string>Cloud</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btn_import">
         <property name="text">
          <string>Import</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_draw">
         <property name="text">
          <string>Draw</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>537</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>slider_threshold</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_threshold</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_proba</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_proba</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_threshold</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_threshold</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>303</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_proba</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_proba</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
